util.yaml

```diff
@@ -3,7 +3,8 @@ globals:
   javascript: |
     // returns the `host` network if running locally otherwise empty string
     function getNetwork(server){
-      return server == 'LOCAL' ? '--network host' : ''
+      // return server == 'LOCAL' ? '--network host' : ''
+      return '--network host'
```

Comment:
What was the reason for changing this? was networking not working as expected in your env?

Reply:
This is a current limitation in the checkpoint/restore with TCP ports that I assume will need to be fixed at some point. As a workaround for now,I changed utils.yaml to always start the SUT services with --network host. Another option would be to use a specific network and give each service a specific IP address.

If I don't do something like this, the restore fails with. I think it's expecting the IP to be the same as it was during the checkpoint.
```
00.019677)    503: inet: 	Restore: family AF_INET6   type SOCK_STREAM    proto IPPROTO_TCP      port 41252 state TCP_ESTABLISHED  src_addr ::ffff:10.16.112.12
(00.019693)    503: tcp: Restoring TCP connection
(00.019698)    503: tcp: Restoring TCP connection id 58 ino 3665db5
(00.019769)    503: Error (soccr/soccr.c:499): Can't bind inet socket back: Cannot assign requested address
(00.019786)    503: Error (criu/files.c:1213): Unable to open fd=27 id=0x58
```